lib/commands/createcache.js

```diff
@@ -79,8 +79,9 @@ function createCache(opts, request, done){
     createCachePayload.expirySettings.expiryDate.value = opts.cacheExpiryByDate;
   }
   if(opts.cacheExpiryInSecs){
-    createCachePayload.expirySettings.timeoutInSec = {};
-    createCachePayload.expirySettings.timeoutInSec.value = opts.cacheExpiryInSecs;
+    createCachePayload.expirySettings.timeoutInSec = {
+      value : opts.cacheExpiryInSecs
```

Comment:
Unnecessary space after `value`.  _(I wasn't going to mention this, but since I had comments for tests, I figured I'd mention it.)_

Reply:
PR updated to fix code formatting issue.